src/module/actor/creature/document.ts

```diff
@@ -778,6 +798,15 @@ abstract class CreaturePF2e<
             focusUpdate.value = Math.clamp(updatedPoints, 0, enforcedMax);
         }
 
+        // Remove special resources from update data
```

Comment:
Are we not able to prevent these from getting included from the template?

Reply:
Its mostly a fail safe in case I missed a spot. It was originally added because of token hud updates, but that's been redirected now. Now it mostly just backhands module devs or temporary breaking changes due to updates.

Ideally I'd prefer to intercept it and do an embedded item update, but embedded item updates triggered as part of an actor update do not go through the _onX pipeline.

Would you prefer if I remove the failsafe? I'm good either way.